airflow/utils/dag_processing.py

```diff
@@ -966,11 +966,31 @@ def _log_file_processing_stats(self, known_file_paths):
         rows = []
         for file_path in known_file_paths:
             last_runtime = self.get_last_runtime(file_path)
+            file_name = os.path.basename(file_path)
+            file_name = os.path.splitext(file_name)[0].replace(os.sep, '.')
+            if last_runtime:
+                Stats.gauge(
+                    'dag_processing.last_runtime.{}'.format(file_name),
+                    last_runtime
+                )
+
             processor_pid = self.get_pid(file_path)
             processor_start_time = self.get_start_time(file_path)
             runtime = ((timezone.utcnow() - processor_start_time).total_seconds()
                        if processor_start_time else None)
             last_run = self.get_last_finish_time(file_path)
+            if last_run:
+                unixtime = int(last_run.strftime("%s"))
```

Comment:
This feels like it should be a log rather than a gauge. Would this not be a time series of timestamps?

Reply:
Correct, it's a time-series of timestamps. Technically this is information is redundant with the `last_run.seconds_ago`, and it is already output in the logs [here](https://github.com/lyft/incubator-airflow/blob/a273afde80432f45329615af77363465bc715314/airflow/utils/dag_processing.py#L1012), so I'm fine to remove this entirely.

Reply:
Updated the PR to remove `last_run.unixtime`. Thanks for flagging it